test/built-ins/RegExp/prototype/test/S15.10.6.3_A1_T22.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2009 the Sputnik authors.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+info: >
+    RegExp.prototype.test behavior depends on the lastIndex property:
+    ToLength(lastIndex) is the starting point for the search, so
+    negative numbers result in searching from 0.
+es5id: 15.10.6.3_A1_T22
+description: "Set lastIndex to -1 and call /(?:ab|cd)\\d?/g.test(\"aacd22 \")"
+---*/
+
+var __re = /(?:ab|cd)\d?/g;
+__re.lastIndex=-1;
+var __executed = __re.test("aacd22 ");
+
+//CHECK#1
+if (!__executed) {
+	$ERROR('#1: __re = /(?:ab|cd)\\d?/g; __re.lastIndex=-1; __executed = __re.test("aacd22 "); __executed === true');
```

Comment:
I'm not a stickler if it's updating an old tests (although this one does add a new one in the old style). Can't hurt though.